projects/wc/src/app/components/generic-ui/detail-view/detail-view.component.html

```diff
@@ -62,9 +63,10 @@
               <ui5-label>{{ viewField.label }}</ui5-label>
               <p>
                 <value-cell
-                [labelDisplay]="viewField.labelDisplay"
-                [value]="getResourceValueByJsonPath(resource, viewField)"
-                [displayAsPlainText]="!!viewField.displayAsPlainText" />
+                [fieldDefinition]="viewField"
```

Comment:
I knew we would get to the point in time that we will have to pass the whole object here ;)